src/java.base/share/classes/jdk/internal/foreign/ImplicitSession.java

```diff
@@ -35,7 +33,7 @@
  * While it would be possible to model an implicit session as a non-closeable view of a shared
  * session, it is better to capture the fact that an implicit session is not just a non-closeable
  * view of some session which might be closeable. This is useful e.g. in the implementations of
- * {@link DirectBuffer#address()}, where obtaining an address of a buffer instance associated
+ * {@link sun.nio.ch.DirectBuffer#address()}, where obtaining an address of a buffer instance associated
```

Comment:
Not sure about this - it is not a case of redundant import - so we should just leave it as is?